server/logs/logs.go

```diff
@@ -0,0 +1,24 @@
+/******************************************************************************
+ *
+ *  Description :
+ *    Package exposes info, warning and error loggers.
+ *
+ *****************************************************************************/
+package logs
+
+import (
+	"log"
+	"os"
+)
+
+var (
+    Info    *log.Logger
+    Warning *log.Logger
+    Error   *log.Logger
+)
+
+func Init() {
```

Comment:
The stock `log.PrintXX` writes to `os.Stderr`. Please keep it this way. Things are going to break if it's changed from `stderr` to `stdout`.

Reply:
Yes, thanks for pointing out. Changed (I'm playing with `stdout` locally).